scripts/run

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+# run the service in one of different deployment modes
+set -e #-x
+
+usage() {
+  cat <<-'EOF'
+usage: run [-h|--help] RUNTYPE
+[-h|--help]       print this help text
+RUNTYPE is one of:
+- tests           run tests against repo
+- integration     run integration tests against running instance;
+                  either `local` or `docker`
+- local           run as a local terminal process
+- docker          run in a docker container
+EOF
+}
+
+# script variables and defaults
+source $(dirname $0)/init.sh
+source "${SCRIPTS_DIR}"/render_configs.sh
+
+JADE_USER_EMAIL=<PUT EMAIL HERE>
```

Comment:
I think the trick is to put the environment variables here? But it doesn't actually pick it up. (It seems like?)

I also can get it to run bootRun here, but not with docker, so that still needs to get fixed.